common-kafka/src/main/java/com/cerner/common/kafka/consumer/ProcessingPartition.java

```diff
@@ -152,6 +152,9 @@ public ProcessingPartition(TopicPartition topicPartition, ProcessingConfig confi
         resetResults();
 
         lastCommittedOffset = getLastCommittedOffset();
+
+        //Consumer fetch cursors might be ahead of last committed offset
```

Comment:
comment formatting consistency
```suggestion
        // Consumer fetch cursors might be ahead of last committed offset
```

Reply:
Done